packages/biometric-ed25519/src/fido2.js

```diff
@@ -0,0 +1,62 @@
+import base64 from "@hexagon/base64";
+
+export class Fido2 {
+    async init(rpId, rpName, timeout) {
```

Comment:
This is an internal class that won't be exposed to public interface level. But I totally agree with you on having named parameters. (Including this, there are lots of places that have funny name/expression because I have ported them from the demo codebase. Due to urgency, I have missed quite a bit but happy to make fixes)